### PR TITLE
Add article about dns issues at Preply

### PR DESCRIPTION
**Pull request by AmetPreply**
Here is a story how did we get issues with DNS because of conntrack.
https://medium.com/preply-engineering/dns-postmortem-e169efd45afd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 A compiled list of links to public failure stories related to Kubernetes.
 Most recent publications on top.
 
+* [DNS issues in Kubernetes. Public postmortem #1 - Preply - blog post 2020](https://medium.com/preply-engineering/dns-postmortem-e169efd45afd)
+    * involved: conntrack, DNS, CoreDNS-autoscaler
+    * impact: partial produciton outage
```
Comment (hjacobs) [inline README.md:8]: small typo

Comment (AmetPreply) [reply README.md]: Nice catch. Thanks!

**Comment by hjacobs** [comment]
Thanks!

I have one question: can you provide details on the action item "Patch prod CoreDNS config"?

BTW: You blog post has some links to `localhost` (e.g. "Kibana link") :smirk:

**Comment by AmetPreply** [comment]
>BTW: You blog post has some links to localhost (e.g. "Kibana link") 😏

Yes, of course, I can't share a link to the private Kibana dashboard, so I just replaced it with localhost.

>I have one question: can you provide details on the action item "Patch prod CoreDNS config"?

I've changed a

```
Patch prod CoreDNS config
```

to

```
Disable autoscaler for CoreDNS
```

**Pull request merged**